nemo_skills/pipeline/score_rm.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Can this file be exactly the same as pipeline/generate.py but only the get_server_command be replaced with get_reward_server_command and inference/generation.py replace with inference/score_rm.py? If that's the case, maybe we can reuse the same entry point and just add a new parameter that controls this or maybe add it as a subcommand with `ns generate rm` whichever is easier to share the code